en/docs/learn/openid-connect-logout-url-redirection.md

```diff
@@ -157,7 +157,8 @@ when you send an OIDC logout request as a POST request:
 ```
 
 For descriptions of all the parameters that you need to specify in the
-POST request, see the <a href="<#parameters>parameter descriptions given above</a>
+POST request, see the <a href="<../../using-wso2-identity-server/openID-Connect-Logout-URL-Redirection#parameters
+>parameter descriptions given above</a>
```

Comment:
this is the wrong link path. this must be the path given in the master branch...this path should be changed to "../../learn/openid-connect-logout-url-redirection#parameters"

Reply:
Fixed